charts/nostr/nostr-multiplexer/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for nostr-multiplexer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: abhsag24/nostr-multiplexer
+  tag: latest
+  pullPolicy: Always
+
+serviceAccount:
```

Comment:
I don't see the serviceAccount resource, are we good to remove from values.yaml and the _helpers?